files.go

```diff
@@ -90,10 +90,11 @@ type File struct {
 	IsStarred       bool     `json:"is_starred"`
 	Shares          Share    `json:"shares"`
 
-	Subject string              `json:"subject"`
-	To      []EmailFileUserInfo `json:"to"`
-	From    []EmailFileUserInfo `json:"from"`
-	Cc      []EmailFileUserInfo `json:"cc"`
+	Subject string                 `json:"subject"`
+	To      []EmailFileUserInfo    `json:"to"`
+	From    []EmailFileUserInfo    `json:"from"`
+	Cc      []EmailFileUserInfo    `json:"cc"`
+	Headers map[string]interface{} `json:"headers"`
```

Comment:
I think it's probably worth doing a Headers type instead which contains:

- `date`
- `in_reply_to`
- `message_id`
- `reply_to`


I know a map[string]interface{} will just catch everything but I'd rather not.